config/configuration.go

```diff
@@ -168,6 +168,7 @@ func init() {
 	viper.SetDefault("pipeline.keyfile", "")
 	viper.SetDefault("pipeline.uipath", "/ui")
 	viper.SetDefault("pipeline.basepath", "")
+	viper.SetDefault("pipeline.signupRedirectPath", "/signup")
```

Comment:
Is it `/signup` or `/ui`. At least this default is different than the config.

Reply:
Sorry, remains of local testing, fixed.